modules/calc/src/main/java/com/opengamma/strata/calc/runner/function/result/ScenarioResult.java

```diff
@@ -34,6 +34,8 @@
 
   /**
    * Returns the result at the specified index.
+   * <p>
+   * The index must be valid, between zero and the number of scenarios.
```

Comment:
This wording is ambiguous. It's only correct if "between" can be interpreted as including the lower bound but not the upper. Which seems a bit of a stretch.